src/index.js

```diff
@@ -315,13 +318,73 @@ async function HandleMessage(message, sender, respond) {
     message = cmd + " " + args;
   }
 
-  if (message == "!open" && sender.isBroadcaster) {
+  if((message.toLowerCase().startsWith("!addalias")) && sender.isBroadcaster){
+    if(message.split(' ').length !== 3){
+      respond("The syntax for adding an alias is: !addAlias command alias, for example: !addAlias open op");
+    } else {
+      let splitMessage = message.split(' ');
+      if(aliases.addAlias(splitMessage[1].toLowerCase(), splitMessage[2])){
+        respond("Alias " + splitMessage[2] + " for command " + splitMessage[1] + " has been added.");
+      } else {
+        if(!aliases.isCommand(splitMessage[1].toLowerCase())){
+          let commands = aliases.getCommands().join(' ');
+          respond("The command entered is invalid. Valid commands are: " + commands);
+        } else if(aliases.isDisabled(splitMessage[1].toLowerCase())){
+          respond("The command " + splitMessage[1] +" is currently disabled.");
+        } else {
+          respond("The alias " + splitMessage[2] + " has already been assigned.");
+        }
+      }
+    }
```

Comment:
I would replace `splitMessage[1]` by `splitMessage[1].startsWith("!") ? splitMessage[1].toLowerCase().substring(1) : splitMessage[1].toLowerCase()` like how it is done for the `!enablecmd`, `!disablecmd`, and `!resetcmd`.
So that `!addalias !weightednext !top` would also work.